services/minespace-web/src/components/dashboard/mine/noticeOfDeparture/NoticeOfDepartureDetails.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Descriptions, Typography } from "antd";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  noticeOfDeparture: CustomPropTypes.noticeOfDeparture.isRequired,
+};
+
+export const NoticeOfDepartureDetails = (props) => {
```

Comment:
why not destructure to individual variables

Reply:
Good thought! Done!